.github/workflows/job-node-publish.yml

```diff
@@ -48,9 +52,11 @@ jobs:
         path: dist
     - name: Publish
       run: |
-        npm version ${{inputs.version}} -m "[no ci] Release ${{inputs.version}}"
+        PUBLISH_DIR="${{ inputs.publishFromDist == 'true' && echo '${{ inputs.workingDirectory }}/dist' || echo '${{ inputs.workingDirectory }}' }}"
+        cd "$PUBLISH_DIR"
+        npm version ${{inputs.version}}
         npm publish
-      working-directory: ${{ inputs.workingDirectory }}
+        cd ..
```

Comment:
or we keep `working-directory: `${{ inputs.publishFromDist? ..... }}` 

and then in - name: Push specify again 
working-directory: ${{ inputs.workingDirectory }}

Reply:
i think it works fine in working-dir, i would prefert to have it in there

Reply:
me2